dmriprep/workflows/base.py

```diff
@@ -293,6 +293,33 @@ def init_single_subject_wf(subject_id):
         ]),
     ])
 
+    if config.workflow.run_reconall:
+        from niworkflows.interfaces.nibabel import ApplyMask
+
+        # Mask the T1
```

Comment:
```suggestion
        # Mask the T1w
```

A very-very stupid nitpick that helps me (and probably only me) sleep a bit better.